### PR TITLE
Add builds checking whether the frontend can be built with all supported nodejs versions

### PR DESCRIPTION
**Pull request by jakubtobiasz**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.12
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | n/a
| License         | MIT

### CODE REVIEW

**Comment by jakubtobiasz** [comment]
The frontend CI does a little bit more than just testing the build process of the assets (it reuses the `https://github.com/SyliusLabs/BuildTestAppAction`), but I don't think it's a big deal. For me, it's better to reuse it than writing it from scratch and maintain it.

**Review by NoResponseMate — APPROVED** [review]

File: .github/workflows/ci_frontend.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+            -   name: Build application
+                uses: SyliusLabs/BuildTestAppAction@v2.0
+                with:
+                    build_type: "sylius"
+                    e2e_js: "yes"
+                    database_version: "8.0"
+                    php_version: "8.1"
+                    symfony_version: "6.4.*"
+                    node_version: ${{ matrix.node }}
```
Comment (NoResponseMate) [inline .github/workflows/ci_frontend.yaml:51]: I just checked the build, can you update the action to cut it short?
I don't see a point in running migrations and fixtures when all we're checking for is whether the assets are getting built correctly

**Review by oallain — APPROVED** [review]

File: .github/workflows/ci_frontend.yaml (hunk 1 above)
Comment (oallain) [reply .github/workflows/ci_frontend.yaml:51]: IMO, the option `e2e_js` also start Chrome Headless and Symfony server, that are not required here. But, `e2e_js` is also required to run `yarn build` which is required.
It's not a blocking factor, but we may have to consider a change in the future.

Comment (jakubtobiasz) [reply .github/workflows/ci_frontend.yaml:51]: The whole workflow takes about 2 minutes. At the same time, we reuse the whole logic, so we keep the workflow as simple as possible. For me, 2m is not a big deal here.

Comment (NoResponseMate) [reply .github/workflows/ci_frontend.yaml:51]: Fair enough

**Comment by NoResponseMate** [comment]
Thank you, @jakubtobiasz!

**Pull request merged**